readthedocs/projects/views/private.py

```diff
@@ -1002,3 +1002,41 @@ def _search_analytics_csv_data(self):
         )
         response['Content-Disposition'] = f'attachment; filename="{file_name}"'
         return response
+
+
+class PageViewAdmin(ProjectAdminMixin, PrivateViewMixin, TemplateView):
+
+    template_name = 'projects/project_page_views.html'
+    http_method_names = ['get']
+
+    def get_context_data(self, **kwargs):
+        context = super().get_context_data(**kwargs)
+        project = self.get_project()
+
+        top_viewed_pages = PageView.get_top_viewed_pages(project)
+        top_viewed_pages_iter = zip(
+            top_viewed_pages['pages'],
+            top_viewed_pages['view_counts']
+        )
+
+        all_pages = PageView.objects.filter(project=project).values_list('path', flat=True)
+        if all_pages.exists():
+            all_pages = sorted(list(set(all_pages)))
```

Comment:
Why are we calling `sorted(list(set` here? Should we not just sort it in the query?